xds/internal/balancer/clusterresolver/resource_resolver_dns.go

```diff
@@ -75,13 +75,15 @@ func newDNSResolver(target string, topLevelResolver topLevelResolver, logger *gr
 	}
 	u, err := url.Parse("dns:///" + target)
 	if err != nil {
-		topLevelResolver.onError(fmt.Errorf("failed to parse dns hostname %q in clusterresolver LB policy", target))
+		ret.updateReceived = true
```

Comment:
We should probably throw a warning log here and in the next error case. I would even be OK with an error log.

Reply:
To keep it consistent with the error logs already in codebase (from ReportError), switched to Info log.

Reply:
Nit: Log lines should be capitalized, unlike error strings.

Reply:
Oh interesting, noted. Switched.